src/main/python/package_bundles.py

```diff
@@ -215,7 +214,7 @@ def prepare_mgmt_bundle(mgmt):
 def prepare_policies(version):
     print("==================================")
     print("Prepare Policies")
-    policies_dist_path = "%s/dist/graviteeio-policies-%s" % (tmp_path, version)
+    policies_dist_path = "%s/dist/gravitee-policies-%s" % (tmp_path, version)
```

Comment:
but perhaps I could rename gravitee- to graviteeio- . So I could keep policies_dist_path named "graviteeio-policies-XXX"

WDYT ?